node_type: eagerx_core.rxbridge/TestBridgeNode
default:
  rate:
  outputs: [tick]
  num_substeps: 10
  single_process: False
  launch_locally: True
  launch_file: '$(find eagerx_core)/launch/rxbridge.launch'
  color: magenta

outputs:
  tick:
    msg_type: std_msgs.msg/UInt64
